Formula/s/stripe-cli.rb

```diff
@@ -0,0 +1,37 @@
+class StripeCli < Formula
+  desc "Command-line tool for Stripe"
+  homepage "https://stripe.com/docs/stripe-cli"
+  url "https://github.com/stripe/stripe-cli/archive/refs/tags/v1.19.5.tar.gz"
+  sha256 "3b59dbc36c66b8021c3c35c1a8d05426f3441c4848b7100469f34360061c1347"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # See configuration in `.goreleaser` directory
+    ENV["CGO_ENABLED"] = OS.mac? ? "1" : "0"
+    ldflags = %W[-s -w -X github.com/stripe/stripe-cli/pkg/version.Version=#{version}]
+    system "go", "build", *std_go_args(ldflags:, output: bin/"stripe"), "cmd/stripe/main.go"
+
+    # Doesn't work with `generate_completions_from_executable`
+    # Taken from `.goreleaser` directory
+    system bin/"stripe", "completion", "--shell", "bash"
+    system bin/"stripe", "completion", "--shell", "zsh"
+    bash_completion.install "stripe-completion.bash"
+    zsh_completion.install "stripe-completion.zsh"
+    (zsh_completion/"_stripe").write <<~ZSH
+      #compdef stripe
+      _stripe () {
+        local e
+        e=$(dirname ${funcsourcetrace[1]%:*})/stripe-completion.zsh
+        if [[ -f $e ]]; then source $e; fi
+      }
+    ZSH
```

Comment:
Wouldn't we be fine if we rename the `stripe-completion.zsh` to `_stripe`?

Reply:
Didn't work for some reason when I first tried it, but it seems to be working now 🤷